README.md

```diff
@@ -0,0 +1,25 @@
+# PostgreSQL
+PostgreSQL is an advanced object-relational database management system
+that supports an extended subset of the SQL standard, including
+transactions, foreign keys, subqueries, triggers, user-defined types
+and functions. This distribution also contains C language bindings.
+
+The latest version of this software, and related software, may be
```

Comment:
Can merge this and the below section and include links to download binaries, instructions to build from source, link to website and docs.